ftw/collectionblock/tests/test_add_collecionblock.py

```diff
@@ -153,6 +149,43 @@ def test_block_amount_limit(self, browser):
 
         browser.login().visit(self.page)
         self.assertEquals(
+            1,
+            len(browser.css('.even'))
+        )
+
+    @browsing
+    def test_is_table(self, browser):
+        create(Builder('sl content page').titled(u'Page 2'))
+        create(Builder('sl content page').titled(u'Page 3'))
+
+        create(Builder('sl collectionblock')
+               .titled(u'A collectionblock')
+               .within(self.page)
+               .having(block_amount=2)
+               .with_default_query())
+
+        browser.login().visit(self.page)
+        # check if there is a table
+        self.assertEquals(
+            1,
+            len(browser.xpath('//table[contains(@class, "sortable")]'))
```

Comment:
The browser supports CSS-Selectors, which would be better readable and safer for this job:
```python
browser.css('table.sortable')
```

The technical problem (appart from readibility) in this situation is that this xpath expression treats `class` as a regular attribute and does not know that it contains space separated classes. So it would also mach `not-sortable` etc. Therefore I prefer using CSS for the simple jobs and use XPath when it gets fancy 😉